.ci/setup.sh

```diff
@@ -77,11 +77,12 @@ else  # Linux
         echo libamdocl64.so > $OPENCL_VENDOR_PATH/amdocl64.icd
     fi
     if [[ $TASK == "cuda" ]]; then
+        echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
         apt-get update
-        apt-get install --no-install-recommends -y curl wget
-        curl -sL https://cmake.org/files/v3.18/cmake-3.18.1-Linux-x86_64.sh -o cmake.sh
-        chmod +x cmake.sh
-        ./cmake.sh --prefix=/usr/local --exclude-subdir
```

Comment:
I'm really glad to see building CMake from source replaced with an installation from a package manager 🙌

Reply:
Huh, I decided that it is enough to have the most recent CMake for our other Linux jobs run inside [Ubuntu 14.04 container](https://github.com/guolinke/lightgbm-ci-docker). Thanks to default CMake from apt-get in Ubuntu 20.04 is quite fresh and passes our minimum version check!
https://github.com/microsoft/LightGBM/blob/ac706e10e461d1c5987a3975cc48aa82fde33566/CMakeLists.txt#L21-L22

As a bonus, we don't need to update it in one more place regularly 🙂